sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/StackFrame.cs

```diff
@@ -10,14 +11,19 @@ internal partial class StackFrame
         /// <summary>
         /// Converts a System.Diagnostics.StackFrame to a Azure.Monitor.OpenTelemetry.Exporter.Models.StackFrame.
         /// </summary>
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode", Justification = "We handle the null condition and call ToString() instead.")]
```

Comment:
I know some of the feedback I had received was to isolate the code producing the warning when suppressing, which could be applicable here as well by moving L19 into its own one-line helper.

Reply:
done